src/lib/aragon-core.js

```diff
@@ -6,16 +6,19 @@ const networks = require('./networks')
 
 IPFS_TIMEOUT = 600000 // 10min
 
-async function getLatestFromRepo (repo, network) {
+async function getLatestFromRepo (repo) {
+  const web3 = new Web3(provider())
```

Comment:
```suggestion
  const web3 = new Web3(provider([
    'frame',
    'direct',
    'wss://mainnet.eth.aragon.network/ws'
  ]))
```

Proposed change from https://github.com/aragon/aragon-desktop/pull/79#issuecomment-536585755

Reply:
Makes sense to me! Thanks @onbjerg